cypress/integration/inline-slots.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { liveblogs, articles } from '../fixtures/pages';
+
+const pages = [articles[2], ...liveblogs];
```

Comment:
Could this be expanded to include any article where `expectedMinInlineSlotsOnPage` exists as a property? I'm worried we'll change the order of the articles in `fixtures/pages.ts` and this test will break

Reply:
Good point! We could filter on articles where that property exists?

Reply:
Good call

Reply:
Nice, thanks for adding @zekehuntergreen